libraries/js/src/types/credential.ts

```diff
@@ -0,0 +1,144 @@
+import { VerifiedEmailAddress, VerifiedGraphKey, VerifiedPhoneNumber } from '../credentials.js';
+import { isArrayOf, isObj, isStr } from './general.js';
+
+interface SiwaResponseCredentialBase {
+  '@context': ['https://www.w3.org/ns/credentials/v2', 'https://www.w3.org/ns/credentials/undefined-terms/v2'];
+  type: string[];
+  issuer: string;
+  validFrom?: string;
+  credentialSchema: {
+    type: 'JsonSchema';
```

Comment:
We don't currently need but may want to allow for `JsonSchemaCredential` as well (a JsonSchema wrapped in its own VC).

Reply:
Yeah. I went fairly strict with static values like this for now. The possibility list is just too long otherwise.